packages/ingest/test/test-S3ProviderClient.js

```diff
@@ -158,3 +159,37 @@ test.serial('S3ProviderClient.sync throws an error if the source file does not e
     }
   );
 });
+
+test.serial('S3ProviderClient.sync syncs a file with an ACL parameter specified', async (t) => {
+  const s3ProviderClient = new S3ProviderClient({ bucket: t.context.sourceBucket });
+  const targetKey = 'someTargetKey1.json';
+
+  await s3ProviderClient.sync({
+    ACL: 'log-delivery-write',
+    destinationBucket: t.context.targetBucket,
+    destinationKey: targetKey,
+    fileRemotePath: t.context.sourceKey,
+  });
+  const objectACL = await s3().getObjectAcl({
+    Bucket: t.context.targetBucket,
+    Key: targetKey,
+  });
+  t.is(objectACL.Grants[0].Permission, 'FULL_CONTROL');
+});
+
+test.serial('S3ProviderClient.sync syncs a file when ACL parameter is not specified', async (t) => {
```

Comment:
I'm not entirely sold that this test is even necessary

Reply:
It looks like the default ACL before this change was `private`. Should we be verifying that in this test?